wsagent/che-core-api-git/src/test/java/org/eclipse/che/git/impl/CommitTest.java

```diff
@@ -111,6 +111,22 @@ public void testAmendCommit(GitConnectionFactory connectionFactory) throws GitEx
         assertEquals(beforeCount, afterCount);
     }
 
+    @Test(dataProvider = "GitConnectionFactory", dataProviderClass = org.eclipse.che.git.impl.GitConnectionFactoryProvider.class)
+    public void testChangeMessageOfLastCommit(GitConnectionFactory connectionFactory) throws GitException, IOException {
+        //given
+        GitConnection connection = connectToGitRepositoryWithContent(connectionFactory, repository);
+        int commitsBefore = connection.log(newDto(LogRequest.class)).getCommits().size();
+
+        //when
+        CommitRequest commitRequest = newDto(CommitRequest.class).withMessage("Changed message").withAmend(true);
```

Comment:
mb it would be better to make some commit in given sesction to make sure that you really change commit message